pkg/eventbus/event_bus.go

```diff
@@ -3,23 +3,34 @@ package eventbus
 import (
 	"encoding/json"
 	"fmt"
-
```

Comment:
@kexun , can we keep go, kubeedge and vendor packages seperated by an empty line ?

Reply:
+1